src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,50 @@
+@import "./utils/_variables";
+
+.nav {
+
+  &__list {
+    display: flex;
+    align-items: center;
+  }
+
+  &__item {
+    list-style: none;
+    margin-right: 20px;
+
+    &:last-child {
+      margin: 0;
+    }
+  }
+
+  &__link {
+    position: relative;
+    line-height: 60px;
+
+    padding: 23px 0;
+
+    text-decoration: none;
+    color: $primary-text-color;
+    text-transform: uppercase;
+    transition: color 0.3s;
+
+    &:hover {
+      color: $primary-color;
+    }
+  }
+}
+
+.is-active {
```

Comment:
let's add active modifier to the links, and move these styles to those modifier
and use is-active class only in html for tests passing